openedx/frontend-app-admin-portal
# feat: Add people management page + zero state

![Screenshot 2024-09-27 at 9 51 00 AM](https://github.com/user-attachments/assets/04083484-0c30-4920-934f-47c91d4a62af)
![Screenshot 2024-09-27 at 9 51 07 AM](https://github.com/user-attachments/assets/ce7a226c-784f-4863-a509-ea0dfffe6a6f)

To test: 
- In EnterpriseAppRoutes.jsx change line 120 to `      {enterpriseAppPage === ROUTE_NAMES.peopleManagement && (`
- in Sidebar/index.jsx, comment out line 158 (the hidden property of the People Management page) 
- run `npm run start:stage` and ensure you can see the People Management tab in the sidebar 
[Jira](https://2u-internal.atlassian.net/browse/ENT-9463) | [Figma](https://www.figma.com/design/eMWRd2jVa4Jqc6y7qrUfaa/Groups-Phase-2?node-id=917-49759&t=S8HT57O4dOVF2cKO-4)

# For all changes


# Only if submitting a visual change
